### PR TITLE
Disneynow

### PR DESCRIPTION
**Pull request by kodmarcel**

### CODE REVIEW

File: streamingscrapper/items.py [hunk 1]
```diff
@@ -31,7 +31,6 @@ class ProgramItem(scrapy.Item):
     series_title = scrapy.Field()
     series_source_id = scrapy.Field()
     series_url = scrapy.Field()
-    season_number = scrapy.Field()
```
Comment (kodmarcel) [inline streamingscrapper/items.py:34]: What changed here and why?

File: streamingscrapper/spiders/disneynow.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+from abc import ABC
```
Comment (kodmarcel) [inline streamingscrapper/spiders/disneynow.py:2]: ??

File: streamingscrapper/spiders/disneynow.py [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+import datetime
+import json
+import logging
+
+logger = logging.getLogger()
+now = datetime.datetime.now()
+BOT_VERSION = "1.0.0"
+
+origin_url = "https://disneynow.com"
+all_shows_url = "https://prod.gatekeeper.us-abc.symphony.edgedatg.com/api/ws/pluto/v1/module/tilegroup/2001767?brand=011&device=001&authlevel=0&&start=0&size=1000"
```
Comment (kodmarcel) [inline streamingscrapper/spiders/disneynow.py:18]: This doesn't seem right

File: streamingscrapper/spiders/disneynow.py [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+
+class DisneynowScrapper(scrapy.Spider, ABC):
```
Comment (kodmarcel) [inline streamingscrapper/spiders/disneynow.py:21]: Why ABC??

File: streamingscrapper/spiders/disneynow.py [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    name = 'disneynow'
+    allowed_domains = ['disneynow.com', 'prod.gatekeeper.us-abc.symphony.edgedatg.com']
+
+    custom_settings = rotating_proxy_custom_settings
+
+    def start_requests(self):
+        yield Request(all_shows_url, method="GET", callback=self.parse_all_shows)
```
Comment (kodmarcel) [inline streamingscrapper/spiders/disneynow.py:28]: This seems like you are getting all fxnetworks shows? is this surely correct api ? are they actually using a same one?

File: streamingscrapper/items.py (hunk 1 above)
Comment (UnknownHero99) [reply streamingscrapper/items.py:34]: season_number was duplicated

File: streamingscrapper/spiders/disneynow.py (hunk 5 above)
Comment (UnknownHero99) [reply streamingscrapper/spiders/disneynow.py:28]: indeed it looks like they are using same api as fxnetworks, only difference is just tilegroup id

File: streamingscrapper/spiders/disneynow.py (hunk 4 above)
Comment (UnknownHero99) [reply streamingscrapper/spiders/disneynow.py:21]: this is a mistake

File: streamingscrapper/spiders/disneynow.py (hunk 2 above)
Comment (UnknownHero99) [reply streamingscrapper/spiders/disneynow.py:2]: this is a mistake

**Pull request merged**